### ISSUE TITLE
Please Discuss Backup of Active Directory

### ISSUE THREAD

**Issue by Mike-Crowley** [MicrosoftDocs/architecture-center#3070]
When deploying AD DS in Azure, how should the AD database be backed up? MABS and MARS explicitly mention support for backing up system state:

- https://docs.microsoft.com/en-us/azure/backup/backup-azure-system-state
- https://azure.microsoft.com/en-us/blog/windows-server-system-state-backup-azure-ga/

However there doesn't seem to be any discussion of the native VM backup and its support for system state / active directory. Do we need to choose "on-premises" backup, even though the VM is in Azure, to protect this workload?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 65f26361-0def-4ae9-e28a-8a8d4f97b87f
* Version Independent ID: ef55eb02-6543-88ad-4f89-5a553848bc67
* Content: [Deploy AD DS in an Azure virtual network - Azure Architecture Center](https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/identity/adds-extend-domain)
* Content Source: [docs/reference-architectures/identity/adds-extend-domain.yml](https://github.com/microsoftdocs/architecture-center/blob/master/docs/reference-architectures/identity/adds-extend-domain.yml)
* Service: **architecture-center**
* Sub-service: **reference-architecture**
* GitHub Login: @telmosampaio
* Microsoft Alias: **pnp**

**Comment by david-stanford** [comment]
Thanks for reporting - this issue is under review. This is a Microsoft Internal DevOps Bug ID # <a href=https://ceapex.visualstudio.com//pnp/_workitems/edit/490221/>490221</a>

**Comment by Justinha** [comment]
@Mike-Crowley Thanks for the question. I want to be sure I follow correctly. Do you mean should you use the built-in Windows Server Backup feature to backup system state for a DC on Azure rather than Azure Backup Service? If so, no, use Azure Backup https://docs.microsoft.com/en-us/azure/backup/backup-azure-system-state.

**Comment by Mike-Crowley** [comment]
I don't believe its supported to backup Active Directory using Azure Backup, if your VMs are in Azure. To protect Active Directory, one must backup "system state" which is not listed unless you select "my workload is on-premises". If I have this wrong, let me know, but either way, it feels like a good topic to discuss in this article.

**Comment by Justinha** [comment]
Yes, agreed. I'm asking experts here and will follow up. The docs I was reading are for on-prem you are right.

**Comment by hallihan** [comment]
@Justinha - did you ever close the loop on this suggestion?

**Comment by Mike-Crowley** [comment]
@hallihan , this is now available in the native azure backup. 
per: 
https://learn.microsoft.com/en-us/azure/backup/active-directory-backup-restore#backing-up-azure-vm-domain-controllers

#If the domain controller is an Azure VM, you can back up the server using [Azure VM Backup](https://learn.microsoft.com/en-us/azure/backup/backup-azure-vms-introduction).

though, yes, it should be mentioned here too.
